src/math/PercentageMath.sol

```diff
@@ -10,8 +10,6 @@ library PercentageMath {
 
     uint256 internal constant PERCENTAGE_FACTOR = 1e4; // 100.00%
     uint256 internal constant HALF_PERCENTAGE_FACTOR = 0.5e4; // 50.00%
```

Comment:
What is the reason behind not using `div(PERCENTAGE_FACTOR, 2)`

Reply:
because Aave is already doing it, so I  guess we can keep it

Reply:
I mean we should at least expose the same function and getters

Reply:
ok ok